app/views/dashboards/analytics.erb

```diff
@@ -2,6 +2,34 @@
 
 <main id="main-content">
   <div class="dashboard-container analytics-container crayons-layout" id="user-dashboard">
+    <% if @organizations.any? %>
+      <nav aria-label="Organizations menu">
+        <ul class="organizations-menu">
+          <li>
+            <a href="<%= dashboard_path %>" class="rounded-btn crayons-link--block rounded-btn--transparent">👈 Back to Main Dashboard</a>
+          </li>
+          <li>
+            <a class="rounded-btn crayons-link--block organization" 
+               href="<%= dashboard_analytics_path %>"
+               aria-current=<%= params[:org_id] ? nil : "page" %>>
+               Your Dashboard
+            </a>
+          </li>
+          <% @organizations.each do |org| %>
+            <li>
+              <a
+                class="rounded-btn crayons-link--block organization"
+                href="<%= dashboard_analytics_org_path(org.id) %>"
+                data-organization-id="<%= org.id %>"
+                aria-current=<%= params[:org_id].to_i == org.id ? "page" : nil %>>
```

Comment:
small refactor (blocking) - we should try to only add this attribute when it's the current page. It doesn't have any real effect on the user, but it causes noise in HTML validators and in the generated HTML generally. Check out this PR that recently cleaned up my work doing the same 😅  #13963

Reply:
Been looking for a way to achieve that. Thanks for sharing those examples ☺️